component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/EventHubClient.java

```diff
@@ -110,24 +103,19 @@ private String getResourceURI(String sasToken) {
             jsonObject.addProperty("applicationId", DataHolder.NODE_ID[i % 5]);
             jsonObject.addProperty("applicationConsumerKey", DataHolder.APPLICATION_CONSUMER_KEY[i % 5]);
             jsonObject.addProperty("applicationOwner", DataHolder.APPLICATION_OWNER[i % 5]);
-            jsonObject.addProperty("apiTier", DataHolder.API_TIER[i % 5]);
-            jsonObject.addProperty("label", DataHolder.LABEL[i % 5]);
-            jsonObject.addProperty("username", DataHolder.USERNAME[i % 5]);
-            jsonObject.addProperty("userTenantDomain", DataHolder.USER_TENANT_DOMAIN[i % 5]);
             jsonObject.addProperty("regionId", DataHolder.REGION_ID[i % 5]);
-            jsonObject.addProperty("apiHostname", DataHolder.API_HOSTNAME[i % 5]);
             jsonObject.addProperty("userAgent", DataHolder.USER_AGENT[i % 5]);
             jsonObject.addProperty("eventType", DataHolder.EVENT_TYPE[i % 5]);
             String payload = jsonObject.toString();
             EventData eventData = new EventData(payload);
             if (!batch.tryAdd(eventData)) {
-                log.info("Sending Events: " + batch.getCount());
+                log.info(("Sending Events: " + batch.getCount()).replaceAll("[\r\n]", ""));
```

Comment:
should this be a debug log?

Reply:
No, I think this handles the case where if some events are unable to be added to the batch, separate batches will be published. The log is required to see how many events were published in each batch, The diff here is a find bug fix.